.github/CONTRIBUTING.rst

```diff
@@ -0,0 +1,69 @@
+##################
+Contributing Guide
+##################
+
+#. Open a Pull Request (PR) to submit any code changes
```

Comment:
General question, do we enforce that no one can push to main directly, they have to create a branch and pass a PR? Right now we can just push directly to main 😨

Reply:
Yup, they will have to. I have disabled pushing to `main` directly; that can be pretty dangerous.